api/app/signals/apps/questionnaires/admin/forms.py

```diff
@@ -35,3 +37,20 @@ def non_related_inlineformset_factory(model, obj=None, queryset=None, formset=No
     FormSet.real_queryset = queryset
     FormSet.save_new_instance = save_new_instance
     return FormSet
+
+
+class QuestionAdminForm(ModelForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if hasattr(self.instance.field_type_class, 'extra_properties_schema'):
+            # Let's add the extra properties validator to the form
+            self.fields['extra_properties'].required = True
+            self.fields['extra_properties'].empty = False
+            self.fields['extra_properties'].validators.append(
+                JSONSchemaValidator(self.instance.field_type_class.extra_properties_schema)
+            )
```

Comment:
This `extra_properties` check is only injected in the question admin. Is the extra properties field on a question meant to store information used for validation?

Reply:
At this moment the extra_properties contain information that can be given to the FE. In this case the `source_url`, it contains the url where the source data of the objects can be found so that it can be drawn on a map etc.

In PR #1055 I am storing a bit more information that is used to determine if multiple answers are given. So please take a look at that too.